lib/ui/common/image_picker.dart

```diff
@@ -0,0 +1,62 @@
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+import 'package:image_cropper/image_cropper.dart';
+import 'package:image_picker/image_picker.dart' as ImagePickerLib
+    show ImagePicker, ImageSource;
+import 'package:intl/intl.dart';
+
+class _Constants {
+  static final EdgeInsets cropAreaPadding = const EdgeInsets.only(top: 20.0);
+  static final Color cropStatusBarColor = const Color(0xffffffff);
+  static final Color cropToolbarBarColor = const Color(0xffffffff);
+  static const double defaultImageRatioX = 1.0;
+  static const double defaultImageRatioY = 1.0;
+}
+
+class _Strings {
+  static final editPhoto = 'Edit Photo';
+  static final noImagePickedError = 'No image picked';
+  static final noCroppedImageError = 'No cropped image';
+}
+
+class ImagePicker {
+  static Future<void> pickImage({
```

Comment:
could return if image was picked or not?

Reply:
👍

Reply:
Done